js_modules/dagster-ui/packages/ui-core/src/sensors/SensorPageAutomaterialize.tsx

```diff
@@ -117,7 +117,7 @@ export const SensorPageAutomaterialize = (props: Props) => {
       return (
         allTicks.map((tick, index) => {
           // For ticks that get stuck in "Started" state without an endTimestamp.
-          if (index !== 0 && !isOldTickWithoutEndtimestamp(tick) && !tick.endTimestamp) {
+          if (!isOldTickWithoutEndtimestamp(tick, index) && !tick.endTimestamp) {
```

Comment:
Same thing re: duplicate endTimestamp check

Reply:
oops this was actually not the same check 😓  so removing it was wrong